android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt

```diff
@@ -1,10 +1,15 @@
 package jp.hacks.smartbread.ui.main.wake.autowake
 
 import android.content.Context
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/90f178290d8f851b5ea294f652a224385d2a1947/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>